src/app/Library/Validation/Rules/BackpackCustomRule.php

```diff
@@ -30,6 +34,12 @@ public static function field(string|array|ValidationRule|Rule $rules = []): self
         $instance = new static();
         $instance->fieldRules = self::getRulesAsArray($rules);
 
+        if ($instance->validatesArrays()) {
+            if (! in_array('array', $instance->getFieldRules())) {
+                $instance->fieldRules[] = 'array';
+            }
+        }
```

Comment:
Could be simplified to:

```suggestion
        if ($instance->validatesArrays() && ! in_array('array', $instance->getFieldRules())) {
            $instance->fieldRules[] = 'array';
        }
```